manifests/d/drakkan/SFTPGo/2.2.0/drakkan.SFTPGo.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: drakkan.SFTPGo
+PackageVersion: 2.2.0
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/drakkan/sftpgo/releases/download/v2.2.0/sftpgo_v2.2.0_windows_x86_64.exe
+  InstallerSha256: 283B84F8A82CE163A0FABF4692C0316FE02668DBF21863AB19C8AFA9A78C7072
```

Comment:
```suggestion
  InstallerSha256: 283B84F8A82CE163A0FABF4692C0316FE02668DBF21863AB19C8AFA9A78C7072
  ProductCode: "{1FB9D57F-00DD-4B1B-8798-1138E5CE995D}_is1"
```

Reply:
Since there's no `Publisher`, we need `ProductCode`, but it's best for you to add `Publisher` to your application.

Reply:
I added the publisher